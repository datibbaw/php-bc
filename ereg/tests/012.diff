002+ abcdefghi
002- abcdef\1ghi