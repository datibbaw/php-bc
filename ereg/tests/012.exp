Deprecated: Function ereg_replace() is deprecated in %s on line %d
abcdef\1ghi