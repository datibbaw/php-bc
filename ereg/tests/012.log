
---- EXPECTED OUTPUT
Deprecated: Function ereg_replace() is deprecated in %s on line %d
abcdef\1ghi
---- ACTUAL OUTPUT
Deprecated: Function ereg_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 135
abcdefghi
---- FAILED
