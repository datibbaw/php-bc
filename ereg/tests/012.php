<?php $a="abc123";
  echo ereg_replace("123",'def\1ghi',$a)?>
