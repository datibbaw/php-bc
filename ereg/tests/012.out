Deprecated: Function ereg_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 135
abcdefghi